pyproject.toml

```diff
@@ -1,3 +1,9 @@
 [build-system]
-requires = ["setuptools", "numpy", "cython", "pybind11>=2.6.0", "wheel"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = [
+"setuptools",
+"numpy; sys_platform != 'darwin'",
+"numpy>=1.20; sys_platform == 'darwin'",
```

Comment:
Maybe keep line 5 and remove `; sys_platform == 'darwin'",`

Reply:
Waiting for your reply on the comment above: Still, I don't understand why we should require `numpy=1.20` (for everyone) if it obviously worked with `1.17`and `1.19` *before* and no changes to the code here have been made.

Reply:
Either both has to be `numpy<1.20` or `numpy>=1.20`.

Reply:
And I wonder how we may specify exactly this?

Reply:
Maybe I'm just too dumb to get the details; shall we have a short meeting on it later? Might be quicker to resolve the misunderstanding that way.